test/GroupByTest.cpp

```diff
@@ -776,24 +764,20 @@ TEST_F(GroupByOptimizations, computeGroupByForFullIndexScan) {
     } else {
       ASSERT_EQ(result.numColumns(), 1);
     }
-    // The test index currently consists of 6 triples.
-    EXPECT_EQ(result(0, 0), idOfA);
-    EXPECT_EQ(result(1, 0), idOfB);
-    EXPECT_EQ(result(2, 0), idOfC);
-    EXPECT_EQ(result(3, 0), idOfX);
-    EXPECT_EQ(result(4, 0), idOfY);
-    EXPECT_EQ(result(5, 0), idOfZ);
 
+    auto getId = makeGetId(qec->getIndex());
+    EXPECT_THAT(
+        result.getColumn(0),
+        ::testing::ElementsAre(getId("<a>"), getId("<b>"), getId("<c>"),
+                               getId("<x>"), getId("<y>"), getId("<z>")));
     if (includeCount) {
-      EXPECT_EQ(result(0, 1), Id::makeFromInt(2));
-      EXPECT_EQ(result(1, 1), Id::makeFromInt(2));
-      EXPECT_EQ(result(2, 1), Id::makeFromInt(2));
-      EXPECT_EQ(result(3, 1), Id::makeFromInt(7));
-      EXPECT_EQ(result(4, 1), Id::makeFromInt(1));
-      // TODO<joka921> This should be 1.
-      // There is one triple added <z> @en@<label> "zz"@en which is
-      // currently not filtered out.
-      EXPECT_EQ(result(5, 1), Id::makeFromInt(2));
+      EXPECT_THAT(result.getColumn(1),
+                  ::testing::ElementsAre(
+                      IntId(2), IntId(2), IntId(2), IntId(7), IntId(1),
+                      // TODO<joka921> This should be 1.
+                      // There is one triple added <z> @en@<label> "zz"@en which
+                      // is currently not filtered out.
+                      IntId(2)));
```

Comment:
At the beginning of this file there is an alias of `I = IntId`, so you can even write
`I(2), I(2), I(2), I(7), ...`